common/journal_entries/imperia_dead_man.txt

```diff
@@ -0,0 +1,109 @@
+﻿je_dead_man_main = {
+	icon = "gfx/interface/icons/event_icons/event_skull.dds"
+	group = je_group_historical_content
+	modifiers_while_active = {
+		dead_man_of_europe
+		country_cannot_enact_laws
+	}
+	immediate = {
+		set_variable = {
+			name = tur_leading_government_progress
+			value = 0
+		}
+		tur_set_dead_man_variables = yes
+	}
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+	}
+	on_complete = {
+		trigger_event = {
+			id = dead_man.2
+		}
+		tur_clear_dead_man_variables = yes
+	}
+	scripted_button = tur_republican_button
+	scripted_button = tur_theocracy_button
+	scripted_button = tur_communist_button
+	status_desc = {
+		first_valid = {
+			triggered_desc = {
+				desc = status_coup_tur
+				trigger = {
+					OR = {
+						dead_man_theocrats_winning = yes
+						dead_man_republicans_winning = yes
+						dead_man_communists_winning = yes
+					}
+				}
+			}
+			triggered_desc = {
+				# Bar not moving, either no supported or not in government
+				desc = status_no_coup_tur
+				trigger = {
+					always = yes
+				}
+			}
+		}
+	}
+	on_weekly_pulse = {
+		effect = {
+			tur_weekly_pulse_communist_progress = yes			#Calculates how much the communists will get next month. Updates weekly, but is not added until the monthly.
+			tur_weekly_pulse_theocracy_progress = yes			#Calculates how much the theocracys will get next month. Updates weekly, but is not added until the monthly.
+			tur_weekly_pulse_republican_progress = yes			#Calculates how much the republicans will get next month. Updates weekly, but is not added until the monthly.
+			tur_leading_government = yes			#The progress bar in this JE is always equivalent to the progress of the dynasty that's currently doing best. This scripted effect determines which dynasty that is, and sets fra_leading_dynasty_progress to be equal to that dynasty's progress variable.
+		}
+	}
+	on_monthly_pulse = {
+		effect = {
+			change_variable = {
+				name = republican_progress
+				add = var:republican_progress_next
+			}
+			#On the monthly tick, the value calculated in the fra_weekly_pulse_republican_progress effect is added to the master progress variable.
+			change_variable = {
+				name = theocracy_progress
+				add = var:theocracy_progress_next
+			}
+			change_variable = {
+				name = communist_progress
+				add = var:communist_progress_next
+			}
+			tur_leading_government = yes			#See above - this ensures that the monthly tick will not momentarily send the status loc out of sync with what the bar is representing.
+		}
+		random_events = {
+			400 = 0
```

Comment:
400 is too little, this should be upped to 800, maybe even 1200.
The events are popping too fast, what's the intended time between them? 
For me, they all popped up over the course of 2~ years, and then I had none of them for the next 4 - 6 years it took me to complete the entry.

Reply:
I've shortened the progress needed to complete, and increased this to 480. But I think I'll raise it higher yea.